src/gui/embed.cpp

```diff
@@ -45,73 +45,70 @@ namespace
 #include "embedded_resources.h"
 
 
-QPixmap getIconPixmap( const char * _name, int _w, int _h )
+QPixmap getIconPixmap( const char * pixmapName, int width, int height )
```

Comment:
Just an opinion, but I tend to prefer `int w; int h` whenever dealing with width and height.  I find it quite intuitive personally.